### DISCUSSION TITLE
How to watch stargazers again - overview page is gone with redesign

### DISCUSSION THREAD

**Discussion by p4cx** [community/community#23629] [Projects and Issues]
<p>As you can see in the picture, Github changed the design of the three buttons in the right upper corner of a repository:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/6/2/62c139c69811a1844b8dc4ffa2e6ce43c27980c6.png" data-download-href="/uploads/short-url/e5CM7OUkzp05AFzd79Y70QqT930.png?dl=1" title="grafik" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/17053360/181078363-b496cac5-2511-49aa-893e-d60f26a1f064.png" alt="grafik" data-base62-sha1="e5CM7OUkzp05AFzd79Y70QqT930" width="690" height="89" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/2/62c139c69811a1844b8dc4ffa2e6ce43c27980c6_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">grafik</span><span class="informations">906×118 7.98 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>The change happened in the last month or so and since then I can’t click on stars to view all the individual stargazers of my repositories. Is there any way for me to get back this functionality?</p>

**Comment by jfullstackdev** [comment]
<p>yup, even me, I cannot click to see the list, as of this moment</p>

**Comment by jsoref** [comment, marked as answer]
<p>On mobile, <a href="https://github.com/e-m-b-a/emba" class="inline-onebox" rel="noopener nofollow ugc">GitHub - e-m-b-a/emba: EMBA - The security analyzer for embedded device firmware.</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/a/a/aaf9d1a8d386e6a61f38b8e7bd79113d9f76bc62.jpeg" data-download-href="/uploads/short-url/oowp5zYVGTejFGvPejGW9JeQ2Lo.jpeg?dl=1" title="Screenshot_20211220-085015" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/2119212/181078364-2d19ca73-680a-42c5-961e-f18bcf0c5317.jpeg" alt="Screenshot_20211220-085015" data-base62-sha1="oowp5zYVGTejFGvPejGW9JeQ2Lo" width="566" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/a/aaf9d1a8d386e6a61f38b8e7bd79113d9f76bc62_2_566x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/a/a/aaf9d1a8d386e6a61f38b8e7bd79113d9f76bc62_2_849x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/a/a/aaf9d1a8d386e6a61f38b8e7bd79113d9f76bc62.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/a/aaf9d1a8d386e6a61f38b8e7bd79113d9f76bc62_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screenshot_20211220-085015</span><span class="informations">1080×954 91 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>The item is above the item you’re used to clicking.</p>
<p>On desktop, it’s on the right side:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/9/2/920d0be5ef51681d1ded97773fc64e43577c0887.png" data-download-href="/uploads/short-url/kQ1F5ahNZBbJb1aV2p0vb4LA4on.png?dl=1" title="Screenshot_20211220-085246" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/2119212/181078365-b81ce718-7083-4dc5-9a3a-5d5684a70f20.png" alt="Screenshot_20211220-085246" data-base62-sha1="kQ1F5ahNZBbJb1aV2p0vb4LA4on" width="676" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/2/920d0be5ef51681d1ded97773fc64e43577c0887_2_676x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/9/2/920d0be5ef51681d1ded97773fc64e43577c0887_2_1014x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/9/2/920d0be5ef51681d1ded97773fc64e43577c0887.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/2/920d0be5ef51681d1ded97773fc64e43577c0887_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screenshot_20211220-085246</span><span class="informations">1080×798 188 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>And it takes you to:</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/e-m-b-a/emba/stargazers">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/e-m-b-a/emba/stargazers" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/2119212/181078366-33ee5741-9676-4534-9579-9f8ec15f287b.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/7/3735fd490be904052f166c1bfae3f446e43661fd_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/7/3735fd490be904052f166c1bfae3f446e43661fd_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/3/7/3735fd490be904052f166c1bfae3f446e43661fd.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/7/3735fd490be904052f166c1bfae3f446e43661fd_2_10x10.png"></div>

<h3><a href="https://github.com/e-m-b-a/emba/stargazers" target="_blank" rel="noopener nofollow ugc">Stargazers · e-m-b-a/emba</a></h3>

  <p>EMBA - The security analyzer for embedded device firmware. - Stargazers · e-m-b-a/emba</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>You could also just URL hack as GitHub urls are regular.</p>

**Comment by p4cx** [comment]
<p>Thank you - this was really helpful. I could have searched for a bit longer, then I could have found this myself.</p>

**Comment by jfullstackdev** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/a/0/a0850432c699ace26ba7b1c69a5977705a58550e.png" data-download-href="/uploads/short-url/mU1nEQxzkC1iuPY8mXOmPyTpoTQ.png?dl=1" title="habol" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47092464/181078367-6bd7452d-9bb9-45aa-868c-8440173e7a07.png" alt="habol" data-base62-sha1="mU1nEQxzkC1iuPY8mXOmPyTpoTQ" width="690" height="388" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/0/a0850432c699ace26ba7b1c69a5977705a58550e_2_690x388.png, https://aws1.discourse-cdn.com/github/optimized/3X/a/0/a0850432c699ace26ba7b1c69a5977705a58550e_2_1035x582.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/a/0/a0850432c699ace26ba7b1c69a5977705a58550e_2_1380x776.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/0/a0850432c699ace26ba7b1c69a5977705a58550e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">habol</span><span class="informations">1920×1080 160 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>it’s because of the mobile version I think, so you need to go to the root of the repo, and click the <code>stars</code> to see them,</p>
